<?xml version="1.0" encoding="UTF-8"?>
<Bucket
   type = "1"
   version = "1.0">
   <FileBreakpoints>
      <FileBreakpoint
         shouldBeEnabled = "No"
         ignoreCount = "0"
         continueAfterRunningActions = "No"
         filePath = "Calculator/CalculatorViewController.m"
         timestampString = "373247554.604463"
         startingColumnNumber = "9223372036854775807"
         endingColumnNumber = "9223372036854775807"
         startingLineNumber = "151"
         endingLineNumber = "151"
         landmarkName = "-undo:"
         landmarkType = "5">
      </FileBreakpoint>
   </FileBreakpoints>
   <SymbolicBreakpoints>
      <SymbolicBreakpoint
         shouldBeEnabled = "Yes"
         ignoreCount = "0"
         continueAfterRunningActions = "No"
         symbolName = "objc_exception_throw"
         moduleName = "libobjc.Y.dylib">
      </SymbolicBreakpoint>
   </SymbolicBreakpoints>
   <ExceptionBreakpoints>
      <ExceptionBreakpoint
         shouldBeEnabled = "Yes"
         ignoreCount = "0"
         continueAfterRunningActions = "No"
         scope = "0"
         stopOnStyle = "0">
         <Actions>
            <BreakpointActionProxy
               ActionType = "Xcode.BreakpointAction.Log">
               <ActionContent
                  message = "Caught here @exp@"
                  conveyanceType = "0">
               </ActionContent>
            </BreakpointActionProxy>
         </Actions>
      </ExceptionBreakpoint>
   </ExceptionBreakpoints>
</Bucket>
